stable/prestashop/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prestashop
-version: 1.0.5
+version: 1.0.6
```

Comment:
the version bump in the mariadb dependency warrants a major version bump in the prestashop chart as it makes the release backwards incompatible.

Reply:
Reverted 9d3a197